tools/python/random_ai/random_ai.py

```diff
@@ -1706,7 +1707,7 @@ def record_create_game(self, button1, button2, max_wins=3, use_prev_game=False):
       custom_recipe_array = [ custom1, custom2, ]
     else:
       custom_recipe_array = []
-    retval = self.player_client.create_game(button1, obutton=button2, opponent=self.opponent_client.username, max_wins=max_wins, use_prev_game=use_prev_game, custom_recipe_array=custom_recipe_array)
+    retval = tools.python.user.create_game(button1, obutton=button2, opponent=self.opponent_client.username, max_wins=max_wins, use_prev_game=use_prev_game, custom_recipe_array=custom_recipe_array)
```

Comment:
This change seems incorrect --- it doesn't work:

```python
Starting game 1
Traceback (most recent call last):
  File "./test_log_games", line 71, in <module>
    c.log_test_game(i, button1, button2)
  File "../random_ai/random_ai.py", line 1802, in log_test_game
    self.record_create_game(button1, button2, use_prev_game=use_prev_game)
  File "../random_ai/random_ai.py", line 1710, in record_create_game
    retval = tools.python.user.create_game(button1, obutton=button2, opponent=self.opponent_client.username, max_wins=max_wins, use_prev_game=use_prev_game, custom_recipe_array=custom_recipe_array)
NameError: global name 'tools' is not defined
```

and it shouldn't be necessary --- `self.player_client` is just as defined as it ever was.

Reply:
I think this was my refactor/rename tooling trying to help me when I moved a directory. a bit frustrating for sure. thanks for bringing it up.